src/screens/APIUtils/APIUtils.res

```diff
@@ -386,6 +386,11 @@ let useGetURL = () => {
         | None => `${userUrl}/${(userType :> string)->String.toLowerCase}`
         }
 
+      // O-M-P Switch
```

Comment:
Can we rename it as Org-Merch-Profile List as there will be different routes for O-M-P switch